params.yaml

```
@@ -0,0 +1,9 @@
+/**:
+  crsf_teleop:
+    ros__parameters:
+      port: /dev/ttyUSB0
```

```suggestion
      port: /dev/ttyUSBPAD
```

and let's change this file to default_params.yaml

---

I have commited this file by mistake, deleting it